packages/carbon-graphs/src/js/controls/Line/Line.js

```diff
@@ -266,7 +266,7 @@ class Line extends GraphContent {
         .select(`g[aria-describedby="${graphData.key}"]`)
         .select(`.${styles.currentPointsGroup}`)
         .selectAll(`[class*="${styles.point}"]`)
-        .data(getDataPointValues(this.dataTarget));
+        .data(getDataPointValues(this.dataTarget).filter((d) => d.y !== null && d.y !== undefined));
```

Comment:
Do we have any utility functions that can we check here whether it is not null/undefined?

Reply:
we do have one with checking null, undefined, balnk(when the value is an object and when the value is a string). Since we only need two from those I did not used that util function.